joint_limits:
  elbow_flex_joint:
    has_velocity_limits: true
    max_velocity: 15
    has_acceleration_limits: true
    max_acceleration: 15
  forearm_roll_joint:
    has_velocity_limits: true
    max_velocity: 15.7
    has_acceleration_limits: true
    max_acceleration: 15
  left_gripper_joint:
    has_velocity_limits: false
    max_velocity: 0
    has_acceleration_limits: false
    max_acceleration: 0
  right_gripper_joint:
    has_velocity_limits: false
    max_velocity: 0
    has_acceleration_limits: false
    max_acceleration: 0
  shoulder_lift_joint:
    has_velocity_limits: true
    max_velocity: 10
    has_acceleration_limits: true
    max_acceleration: 10
  shoulder_pan_joint:
    has_velocity_limits: true
    max_velocity: 12.56
    has_acceleration_limits: true
    max_acceleration: 15
  torso_lift_joint:
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 5
  upperarm_roll_joint:
    has_velocity_limits: true
    max_velocity: 15.7
    has_acceleration_limits: true
    max_acceleration: 15
  wrist_flex_joint:
    has_velocity_limits: true
    max_velocity: 22.6
    has_acceleration_limits: true
    max_acceleration: 25
  wrist_roll_joint:
    has_velocity_limits: true
    max_velocity: 22.6
    has_acceleration_limits: true
    max_acceleration: 25
